broker_app.py

```diff
@@ -193,5 +194,29 @@ def get_schemas():
     return response_json(HTTPStatus.NOT_FOUND, None)
 
 
+@cross_origin()
+@app.route("/spreadsheets")
```

Comment:
I understand the purpose of using blueprints is to help organise and maintain the routes. I didn't add the route as a blueprint since there was only one route being added.

Could you please explain why it's better adding it as a blueprint? Are we anticipating adding more routes related to geo accessions / geo_to_hca?

Reply:
I don't know if we add more routes, but it would be better to keep the broker_app module clean, and give each file/module/unit a single reason to change.

Reply:
about the route name, the full url is currently: `<broker_url>/geo-accession/spreadsheet?geo-accession<geo_accession>`.

I am considering changing it to `<broker_url>/pre-filled-spreadsheet/geo-accession/<geo_accession>`. Would you have any feedback / suggestion for this?

Reply:
This should follow the REST API principles, which is about resources.
The resource here is the import.
The API should be:

```
POST /import-geo?accession=1234
```

or a slightly more general
```
POST /import?source=geo&id=1234&id-type=accession
```
 
This way it would support a synchronous operation mode where the spreadsheet is returned as well as an asynchronous mode where an id is returned that could be created later:

```
GET /import?id=fab123/status
```
which would return whether it is running, failed, finished.

```
GET /import?id=fab123/spreadsheet
```
which would return the spreadsheet

Reply:
I don't understand why this should be a `POST import` call. We aren't importing anything into the system, we are exporting a spreadsheet for download however, in which case, a `GET` call with `export` somewhere in the url makes more sense.

Reply:
@yusra-haider The POST is because we are creating a new job to import data from geo.

Reply:
I don't understand or agree. We're getting / downloading a spreadsheet from the UI. We aren't import or creating any new resource in ingest; why would it be `POST` then?

Reply:
Had a discussion with @amnonkhen. 

The reason for making this a `POST` call is because we're `posting` a job to ingest to get data using geo accession. Wrt `export` vs `import`, we decided to go with `import`, as we ultimately plan on using this endpoint to create / import data directly in ingest (instead of returning a spreadsheet)